beacon_node/beacon_chain/src/attestation_rewards.rs

```diff
@@ -134,10 +133,8 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         let spec = &self.spec;
 
         // Calculate ideal_rewards
-        let participation_cache = ParticipationCache::new(&state, spec)?;
-        process_justification_and_finalization(&state, &participation_cache)?
-            .apply_changes_to_state(&mut state);
-        process_inactivity_updates(&mut state, &participation_cache, spec)?;
+        process_justification_and_finalization(&state)?.apply_changes_to_state(&mut state);
```

Comment:
`process_justification_and_finalization()` requires the `progressive_balances_cache` to be initialized but from what I can see it never is since this state is loaded from the database before being passed to this function.

Reply:
good catch, added some more initialisation in https://github.com/sigp/lighthouse/pull/5386